udf-examples/pom.xml

```diff
@@ -115,6 +115,40 @@
             </dependency>
         </dependencies>
     </profile>
+    <profile>
+        <id>release311cdh</id>
+        <activation>
+            <property>
+                <name>buildver</name>
+                <value>311cdh</value>
+            </property>
+        </activation>
+        <dependencies>
+            <dependency>
+                <groupId>org.apache.spark</groupId>
+                <artifactId>spark-sql_${scala.binary.version}</artifactId>
+                <version>${spark311cdh.version}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>org.apache.curator</groupId>
+                        <artifactId>curator-recipes</artifactId>
+                    </exclusion>
+                </exclusions>
```

Comment:
If we're pulling in the CDH version of the Spark jars, why would we need to override a dependency and replace it with something else?  Are the dependencies wrong on their jar?

Reply:
I assume so, this is what they had in the shim layer that they wrote.. I'll follow up with them to see if we can remove later